.travis.yml

```diff
@@ -1,6 +1,17 @@
-language: node_js
-services: mongodb
-sudo: false
-node_js:
-  - "8"
-  - "10"
+language: node_js
+services: mongodb
+sudo: true
+before_install:
+  - sudo apt-get update -q
+  - sudo apt-get install python3-pip python-dev -y
+  - pip3 install -U setuptools
+  - pip3 install -U virtualenvwrapper
+install:
+  - nvm install-latest-npm
```

Comment:
Why is this added? Are the versions for `node_js` ignored now?

Reply:
This is because the `npm` used to install packages was not working for `webgme-engine` installation. The error I got was (no module named `text`)

Reply:
gotcha. Why not `npm install -g npm`?

Reply:
Our build with different `node` versions are managed using `nvm` by travis. I think we can certainly replace this with `npm install -g npm`.

Reply:
https://travis-ci.org/deepforge-dev/deepforge-keras/builds/608312198. 
Travis build is passing with `npm` update using `npm`.